### DISCUSSION TITLE
Nested selector like vanilla CSS

### DISCUSSION THREAD

**Discussion by zkkzkk32312** [tailwindlabs/tailwindcss#13858] [Ideas]
Hi there,
Is there a way I can achieve the following vanilla CSS functionality with tailwind's inline styling approach? Note in this nested selector, classB can be N level's below classA.

.classA .classB {
    /* whatever styling goes here*/
}

<body class="classA">
    <div>
          <div class="classB">

**Comment by andymccray** [comment]
Yep, if you need that level of specificity you can do something like this:

https://play.tailwindcss.com/L7LMWkMDMq

**Reply by zkkzkk32312** [reply]
Hey thanks for the quick reply, 

I assume that approach would actually require every element in between to have a class to reference to?
in the vanilla CSS, the selector is much looser, and the selector simply doesn't care what classes they have or even have a class at all.

**Reply by andymccray** [reply]
Only if you made that class target direct children (e.g. `[&_.classA_>_.classB]`). This should work just fine:

https://play.tailwindcss.com/jenldlBR6u
